bin/commands/components/disable/index.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project.
+#######################################################################
+
+print_level1_message "${ZWE_CLI_PARAMETER_CONFIG}" "${componentCfgPath}.enabled" "false"
+require_zowe_yaml
+
+componentCfgPath=
+componentCfg=
+if [ -n "${ZWE_CLI_PARAMETER_HA_INSTANCE}" ]; then
+  componentCfgPath="haInstances.${ZWE_CLI_PARAMETER_HA_INSTANCE}.components.${ZWE_CLI_PARAMETER_COMPONENT_NAME}"
+else
+  componentCfgPath="components.${ZWE_CLI_PARAMETER_COMPONENT_NAME}"
+fi
+componentCfg=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".${componentCfgPath}")
```

Comment:
Similar to other comment, think we should use `find_component_directory` to check if the component is installed.